frontend/src/scenes/plugins/PluginImage.tsx

```diff
@@ -0,0 +1,40 @@
+import { Card } from 'antd'
+import { parseGithubRepoURL } from 'lib/utils'
+import React, { useEffect, useState } from 'react'
+import imgPluginDefault from 'public/plugin-default.svg'
+
+export function PluginImage({ url }: { url: string }): JSX.Element {
+    const [state, setState] = useState({ image: imgPluginDefault })
+
+    useEffect(() => {
+        if (url.includes('github.com')) {
+            const { user, repo } = parseGithubRepoURL(url)
+            setState({ ...state, image: `https://raw.githubusercontent.com/${user}/${repo}/main/logo.png` })
```

Comment:
This is cool! Though we should probably save this into the plugin model somehow and give an option to specify a custom logo via a property in `plugin.json`. Totally fine to release this as is though.